apps/system/js/notifications.js

```diff
@@ -641,6 +642,11 @@ var NotificationScreen = {
 
   updateNotificationIndicator: function ns_updateNotificationIndicator() {
     if (this.unreadNotifications.length) {
+      // If the ftu is running we should not show the ambient indicator
+      // because the statusbar is not accessible
+      if (FtuLauncher.isFtuRunning()) {
```

Comment:
you should be able to use `Service.query('isFtuRunning')` here (more future-proof :))